action.yml

```diff
@@ -32,6 +36,10 @@ runs:
       shell: sh
       if: runner.os == 'macOS'
       run: |
+        for f in $(gchmod -c -R +r . | awk '{print substr($3, 2, length($3)-2)}')
```

Comment:
`gchmod` isn't available on _my_ Mac. 😅 

Is it available on the standard Actions macOS runner image? And, do we need to worry about self-hosted runners as well? 😬

Reply:
Oh, I should check. It's available on my mac and I didn't install it (or don't remember doing it). As for windows, ntfs file permissions don't map directly to linux (unix? posix?) permissions, so I left them unchanged.

Reply:
`gchmod` probably came from homebrew, but for for the life of me I couldn't tell you which formula installed it because it wasn't by me. I reverted to using `chmod` on mac, which simplifies the string parsing anyway.